cip/1.accepted/CIP2018-05-03-catalog-administration.adoc

```diff
@@ -0,0 +1,284 @@
+An error is raised if `<nested subquery>` does not return a graph.
+
+3. `CREATE GRAPH myProc(...) AS <graph name>` creates a new graph in the catalog that is a copy of the graph that was returned by the call to `myProc`.
+An error is raised if `myProc` does not return a graph.
+
+4. Multiple graphs may be created with one graph statement by separating multiple `GRAPH` subclauses of the forms 1, 2, or 3 with `,` (e.g. `CREATE GRAPH foo, GRAPH bar { ... }`)
+
+5. An error is raised when a graph is created with the name `<graph name>` and there already is a graph with that name in the catalog.
+
+
+=== Delete graph
+
+The catalog command `DELETE GRAPH <graph name>` deletes the graph with the name `<graph name>` from the catalog.
+If `<graph name>` is not the name of a graph that already exists in the catalog, an error is raised.
+
+
+=== Copy graph
+
+The catalog command `COPY <old name> TO <new name>` copies the content and the schema of the graph with the name `<old name>` in the catalog to a new graph with the name `<new name>` in the catalog.
+If `<old-name>` is not the name of a graph that already exists in the catalog, an error is raised.
+If `<new-name>` is the name of a graph that already exists in the catalog, an error is raised.
+
+
+=== Rename graph
+
+The catalog command  `RENAME <old name> TO <new name>` removes the graph with the name `<old name>` from the catalog and adds it as a new graph with the name `<new name>` in the catalog.
+If `<old name>` is not the name of a graph that already exists in the catalog, an error is raised.
+If `<new name>` is the name of a graph that already exists in the catalog, an error is raised.
+
+
+=== Truncating graphs
+
+The catalog command `TRUNCATE <graph name>` truncates the graph with the name `<graph name>` in the catalog.
+
+Truncating a graph deletes all its nodes and relationships but retains any additional schema information like constraints.
+
+
+=== Determining the name of a graph
+
+The `catalog()` function returns the catalog name for the working graph or `NULL` if the working graph is a dynamically constructed graph.
+
+The `catalog(g)` function returns the catalog name for the graph identity `g` or `NULL` if `g` is a dynamically constructed graph.
+
+Note:: `toString(graph())` may be used to just generate a human readable name for the working graph.
+
+
+== Views
+
+A view is a query that is stored in the catalog (in the same way as graphs) but is re-evaluated whenever it is referenced.
+This is called an activation.
+
+
+=== Creating views in the catalog
+
+Views are created in the catalog using syntax `CREATE QUERY <view name> { <nested subquery> }`.
+
+Multiple views and graphs may be created by one `CREATE GRAPH/QUERY` schema command.
+
+
+=== Managing views in the catalog
+
+Views in the catalog can be managed with `COPY`, `RENAME`, `DELETE QUERY` in the same way as graphs.
```

Comment:
Why the distinction that `COPY` and `RENAME` do not require the additional keyword of what is copied/renamed, but `DELETE` needs that?